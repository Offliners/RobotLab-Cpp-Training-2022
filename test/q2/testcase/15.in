874
ELEVATOR
OBJECT
cAFe
aLteRnAtIVELy
BErry
FaiTHfUL
scOrN
VaLlEY
act
honorAble
ASS
FREEZER
SHElter
tractor
65567690AR7
kIdNEY
9A97
PERiOdIC
AiR9T9g69
StEep
MaStEry
65N
intRiguE
certIfIcAte
DISQUALiFY
inefFiCiEnT
aPprovE
CoMPLY
ForMatiOn
pROSpERITY
mouse
ARCHIVE
ABorIgine
CRISIS
DISCONNECT
OXIDATION
DEceptiOn
CoNtAIn
TENnIs
DESIRE
CuStodY
alienation
descriPtion
teNSe
PAiNTiNG
scrAtcH
FlOWER
BBC
SLOW
continuouS
mars
adjust
HIrE
NI54
TORTOISE
owL
impLY
0i5e5
haRneSs
FLOW
dENoTE
SHake
9o38O4
OfFend
06864i6
dutiful
ClOthe
victoRy
nOvelty
TREsPASS
lIvESTOCk
ThUn7ER
SUCK
ReTOrT
40SpeNsIOn
lIfeGuarD
SUGAr
54M5oN3
UPGrADE
But7o7
ACCELERATION
port
PIGeon
l
N116eC4
GIFTED
StuPId
swaN
lIGHtning
547s2
PerfectiON
oMeN
HARDEN
MUdDy
wEddINg
thorough
sUPERHiGHWaY
termInaTE
ARrOGANT
head
secrEtAry
dRuNKarD
6ri6nta9
cleanINg
GrADUaL
stRanGer
ACcordIngly
Dilute
oppRess
ARCHITECT
destructive
eXAMinEE
CHUNK
SIN
canal
SINCeRiTY
StRoNG
pATRon
bLaZe
CONFER
99EN7n3
DOMINANT
CHERRY
JEANS
PaTHETiC
47RA636
HINDER
dODgE
fRuIT
meChanics
nEGotiatE
FErTiLiZeR
sEven
SOMEBoDY
DEaF
MOBILe
bRoAD
u8De75561311E9
AscRIbE
WE7NES714
MONOtONY
SOUTHERN
v
additional
ResidE
InteGral
HAIRDRESSER
shORE
aDvIsORy
conscious
70OOp
OATH
condoM
dEaFEN
delicate
further
ClIMaTE
bASS
Used
steward
FLOOd
LOss
EcONOmY
SNATCH
Second
INDIGNATIoN
distInCT
MiSSIonARy
away
recIPieNt
8V8956
OFFICER
MaLICIOUS
fURIOuS
DESpERATE
Assistant
miNute
AtteNtIon
asSURancE
drown
craM
GradUATION
dEscendaNt
AWful
RegULAR
0Anac76
ignore
05W
FoG
POSTpONE
GreEd
GREEN
AIRWAY
PREDICAmENT
AWaIT
CHaLlengE
FulL
URgeNcy
prUdent
asSigN
sHow
BUTTERFly
3uM90I4
VOLUnTaRy
mOmenTUM
SIZAbLe
lItTEr
rally
UNFaIR
885iolo8Y
reNEw
cap
BAMBOO
5E59
nation
ReAD
dELIBeration
iLl
LIE
INitIALLY
dISAPPEAr
coNvInCiNG
tempt
DIReCTIOn
SITUATED
Close
sovereIgn
CoMrADE
eNEMY
RATIoNAL
O
tWENt5
BecKOn
COAst
TaNkER
Motif
LOudspeAKeR
TRIUMPHANT
before
IMMORAL
5M0A8S9
c6n4R3134at8
whERe
RelieVE
silly
despISe
pULMONARY
STRoKe
art
EVEN
ADVanCE
LaUGH
INVESTIGATE
speed
lOYALTY
SHEEP
EMOTION
idenTITY
PROBLEM
dISc/DISk
ruDe
GRAMMaR
dignitary
DeepEn
mIlitaRy
UPROOT
uMbrella
ac1uaintan8e
KIDnAP
slide
TexTbOoK
IMmIGRaNt
pICK-UP
eXpLaIn
4256R3D
PREFERENCE
wEAken
cReatE
hEAp
separaTion
rANK
COMPAsSIOnAtE
TWIg
DoOr
shockEd
DEAn
gRanD
vACuum
EXCellENt
FrOsT
contRIBuTIoN
CUsTOMeR
SupErStitiOuS
StALl
T44
BOuNCe
gUe0s
coNVINCE
GARBAGE
9Oti3E2b46
alphabet
plug
band
flour
mill
coastguaRd
LeaP
discuss
nEt
WhITE
29496t0L1
9or2u29
hIJACK
fEATURe
fUr
HEIGHT
LoOM
SIgNatURe
CATalOguE
mi12S07y
beyOnd
Shop
mERCY
CURIOSITY
solVe
Se9
LEMonaDe
PLENTIFUL
DISsIDEnT
NUisANce
EdUcaTeD
librAry
FuturE
POp
DEFENd
tUB
psychOlOgy
62630t6on
BIrd
launDry
KINDerGARTeN
1E3
consent
NEWCOmEr
CONSTITuTIONAL
TelegRam
LIGhTWEIGHT
personal
FAST
eMbargo
MemorIAL
bay
r683r167
SUpERInTenDeNt
2aZz3e
57plY
NoD
04b17
aPPEtitE
ClEAnER
PERSUASive
coMpetENce
s4g4al
BIolOGIcAL
IgNoraNt
anyhow
IMpACt
sAndwich
cop
VoTEr
oystEr
BiTtErly
eViDENt
perch
evAluation
WhoEVer
Dough
silent
hygiEne
t
litERate
RADIATE
COUNSELLOR
MIserY
hardLy
hiErarCHy
BritoN
aPpRopriate
cOOrdiNatiOn
fortune
STATE
bAI4
GuIdeline
AwKWARDLY
battLe
ingredIeNt
cOme
delicAcy
diSease
p
doubtless
dAILy
t3BErCuLoSIS
cULtuRal
1ECON3ILE
own
EXPEL
iNteRseCTION
CaShI7R
twin
pRIDe
tuMOr
gRoUp
biNgo
RAId
LACE
TUToR
ninEtEeN
TASK
tOothPAStE
outstandiNg
DonatE
BASIC
StandPoint
3lOPPY
PROfESSor
dRagoNfLy
GREENHOUSe
ProVINCIaL
fALl
like
BLANk
content
thINk
MILLIGRAM
MODERNIZATiON
forecast
SNaP
elEganT
tHunder
tempLe
DANCe
aSpiRaTIoN
caress
PROBABLE
prefacE
Riddle
anT
WEB
cHAmber
AvENuE
PO8I46
StabIliTy
BATHTUB
EVIL
GULF
ENABLE
bAptiSM
SCaRF
RESIST
Groom
127F
eXpecT
ExTENSIVe
DEsiGnEr
SunshiNe
fRAmE
GRIeVe
us
GoOdnesS
iNhabiTanT
jUnk
quArteR
feet
ACTuAL
CArROt
downwa67s
6ro70
EIGHTY
SadDle
accessory
D45676n424
FLOCK
economicaL
hOu20W1F9
BRACE
MISSION
4U4I82T7
AIRC41Ft
CUlT
SkIRT
REBEL
OrDInARIlY
resistance
pioneeR
03t71g4
applAuSE
WIDEN
aGGrEgaTe
wheelchaIr
policy
teach
bravery
cluMSY
coMFOrtaBLY
DISadvANtAGe
TRAvEl
Panic
ItCH
UNPReDICtAble
hAndBoOK
HuMILIATe
CLUSTER
2O3ple
DRAWBaCK
wEDnEsday
CALL
FlOaT
courteoUs
hypocrisy
PINEAPplE
behaVe
instrucTIOn
park
restroom
analogy
2LSO
BECoME
m9tt5R
bAnkRUpt
per
wait
pOWDER
data
TuMPIKe
dIstress
cAPTure
bLoc
ASHAmED
INTERPRETER
greet
33T2RGEnT
failure
asSOciaTE
ministeR
marGINAL
REPUbLIcaN
DIversiFY
compLete
OUT-OF-DATE
en813A3c0
HaiRStyLE
MORTAL
invisibLE
Contrary
maid
GaP
pedesTrian
MAKE
Bat
MAYOR
PERseverANcE
NiGHTMaRe
telephone
Female
O21L4W
REpOrTer
dUTy
iRRitABLE
RElEasE
cUrIOUs
SHIFt
MisTAkE
LiFe
Click
0OAM
ENtHUSIASTiC
theo2y
OPeRatiOnal
NiCkNAME
DIFfErENTiAL
rAy
TRial
diVidE
UNiT
HYPOCRITE
8u071r
hOMelANd
ELSE
goldEn
atTaiNmenT
PROUd
stORAgE
teENaGer
pArk
recORd
abide
ENhAN0E
sINg
hypothesis
HUMBlY
coInciDencE
amuse
ChILDlikE
AUTOBIOGRAPHY
Oven
ethiC
tREAsURE
PolE
pawN
staY
Stir
socIallY
judge
negligiBLe
cHanCe
SWEeT
NePHew
DRaW
visIBle
inDiFFerEnt
barber
PoTAtO
agent
pITCHER
EARN
ADAPTATION
cOmplICATE
DrOpoUT
PATteRn
MUSeUM
BaRTEndEr
aMazE
BoOM
IrResPoNsIbLe
NEgLECt
REMAIn
ROLL
IMAGe
melancholy
DIAmoNd
REHEARSe
OWNERShIP
929091n453
mAnNeRs
STRAin
7243ilY
UNIVERSITY
linguistics
DemoCrAcY
SEASon
1NITIAtE
hUMilIate
c87R05965O7
FEVER
evolution
ENvIROnMENT
cRouCH
NeWscAsTer
ONCE
beEf
DRAiNAGE
GO
beijing
defInITe
habit
finiSH
StRAw
conScientious
35278h
DEnTAl
type
WhaTEVer
pLASTER
EXercIsE
cOcKrOacH
IDeNTIFicaTION
iNvent
PORtER
JUMP
PEnnY
LAP
tabloid
smooth
O28756132
sUrface
aLley
select
HaPPINESS
NinetY
commuNiCATion
oBsTAcLE
paInFully
rEsultant
cAuse
barbariaN
HOWl
AMAzinG
607646551I30
ECOnomIcS
BrOwN
ELEcT
nAp
INTerCHANgEABlE
PRiVATeLY
TOUCHSTONE
QUarrEl
elEcTRIc
mAGnEt
array
7om7634s
LIBERAL
honey
company
tWINkLE
WAV7
political
d0m4cr52
MANY
pRINCIpALlY
o
cAnDlE
cAn
somedaY
RELUCtANTLy
Country
GEneration
RevolUTIOn
iNtelLIGenT
crowdEd
holiday
dismantle
outdoor
INtEREST
AtTRaCTION
RESERVATION
cLUtcH
iNDoOrS
DOWNsTaIRS
visItor
SUNLIgHt
eSsenTial
FaNcY
BOXING
SPONsOR
relatively
HeADiNg
SIMiLArLY
ITALIAN
NAG
AlMost
ENGINEER
DeFENSIVE
BiOcHeMISTrY
reSponsIBilITY
lingering
DEpICT
aPpAReNt
ACTIviTY
RESOLUtIoN
ABiLITY
eMiGRANt
EVERy
HUMBLY
King-siZE
indoOr
PILE
recYcle
consumption
optimISm
INDUSTRIOU4
SlEEP
menu
comPETitOr
ARm
ENougH
C6STL7
n
D5d86
viaBlE
railway
diVe
opPonenT
ECho
MeN
COCKTAIL
soIl
A0ce9ER04io7
TAG
CONSuLT
ABLE
CoMMENTAtOR
GalLOn
any
ExIT
Divine
hosT
boast
BREAK-UP
tYPe
COluMNisT
drAmA
diZZy
exHIBit
8eBELlio1
pizza
INSURaNce
dESsert
gIFT
HOT
CLay
BULly
SoRe
Magnetic
Swing
PleDGe
Honor
aDvertising
SmAlL
citIZEn
32l3339
PasSaGe
alLot
sew
conVict
MEANING
banQuet
DEAL
cirCuLatE
DISORDER
VIOlENCE
BAcON
ofFsPRing
Queue
dIGITal
jET
comPlaint
GoVerNmenT
hurRICane
EDGE
SPIT
DEMOnStRAtE
pOse
rEceiPt
rAZoR
IlLeGAL
DRAFT
8r14s1975a4io2
erosiON
a1d9t8o80l
8Nvi6aT0O1
liaR
5393560979T259
MIllIoN
SeNtEnCE
Dignity
ENgiNEERInG
pEACH
huMorouslY
RipE
CandIdaTE
cASHIEr
gueSs
CRYSTAL
caSh
admiraL
HandSOMe
flee
haRD
whale
eqUALity
INTEGrITY
TradiTiON
TRANQUiL
