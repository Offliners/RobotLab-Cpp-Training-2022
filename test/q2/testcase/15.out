n. 電梯；升降機
n. 物體；目標
n. 咖啡店；咖啡
adv. 二擇一地
n. 果
adj. 忠誠的
n./v. 輕蔑；鄙視
n. 山谷
n./v. 行動；表演
adj. 榮譽的
n. 驢；蠢人
n. 冰箱
n./v. 避難所
n. 拖拉機
Unknown
n. 腎
Unknown
adj. 定期的
Unknown
adj. 陡峭的
n. 控制；熟練
Unknown
v./n. 密謀；陰謀
n. 證書
v. 取消資格
adj. 無效率的
v. 同意；認可
v. 依從
n. 形成；組成
n. 發達
n. 鼠；滑鼠
n. 檔案
n.  原著居民
n. 危機
v. 使分離
n. 氧化(作用)
n. 欺騙
v. 包含
n. 網球(運動)
n./v. 慾望
n. 監管
n. 離間；疏遠
n. 描寫；說明?
adj. 緊張的
n. 圖畫
v. 抓(傷)
n./v. 花朵；開花
n. 英國廣播公司
adj. 緩慢的
adj. 連續的
n. 戰神；火星
v.(n. )調整
v./n. 租借；雇用
Unknown
n. (陸)龜
n. 貓頭鷹
v. 暗示
Unknown
n. 馬具
n./v. 漲潮；流動
v. 意味著
n./v. 搖動
Unknown
v. 冒犯；觸怒
Unknown
adj. 守本分的
v. 為…穿衣
n. 勝利
n. 新鮮；新奇
v. 擅入
n. 家畜
Unknown
v. 吸；吮
v./n. 反駁
Unknown
n. 救生員
n. 糖
Unknown
v. 提升
Unknown
n. 加速
n. 港
n. 鵲子
Unknown
Unknown
adj. 有天賦的
adj. 愚蠢的
n. 天鵝
n. 閃電；閃光
Unknown
n. 完備
n. 微兆；預兆
v. 使變硬；使堅強
adj. 泥的；骯髒的
n. 婚禮
adj. 徹底的
n. 超級高速公路
v. (使)終止
adj. 傲慢的
n. 頭
n. 秘書
n. 酒鬼
Unknown
n. 掃除
adj. 逐步的；平緩的
n. 陌生人
adv.  因此
v./adj. 稀釋(的)
v. 壓迫；壓制
n. 建築師；設計師
adj. 毀滅性的
n. 應試者
n. 厚塊
n./v. 罪惡
n. 運河
n. 真誠
adj. 強壯的
n. 贊助人
n. 烈火；光輝
v. 商議；授予
Unknown
adj. 支配的
n. 櫻桃
n. 牛仔褲
adj. 悲哀的
Unknown
v. 妨礙
v./n. 避開
n. 水果；成果
n. 力學；機械學
v. 談判
n. 肥料；促進發展者
adj./n. 七
pron. 有人
adj. 聾的
adj. /n易；汽車
adj. 寬廣的；概括的
Unknown
v. 把…歸因於
Unknown
n. 單調
adj. 南方的
Unknown
adj. 附加的
v. 居住
adj. 構成整體所需要的
n. 美髮師
n. 岸；濱
adj. 提供忠告的
adj. 神智清醒的
Unknown
n. 誓約；詛咒
n. 保險套
v. 使聽不見
adj. 纖弱的；雅緻的
adj./adv./v. 進一步的
n. 氣候；風氣
n. 低音部
adj. 舊的
n. 服務員
n./v. 洪水；淹沒使氾濫
n. 喪失；遺失
n. 節約；經濟
v. 搶
adj./n.第 v. 的；秒
n. 憤慨
adj. 有區別的
n. 傳教士
adv. 離開；不在
n. 接受者
Unknown
n. 軍官；官員
adj. 存心不良的
adj. 狂怒的；猛烈的
adj. 絕望的
n. 助理
n. 分鐘
n. 注意力
n. 保證
v. 把…淹死；淹沒
v. 填塞
n. 畢業；畢業典禮
n. 後代
adj. 糟透的
adj. 整齊的；規律的
Unknown
v. 不顧；忽視
Unknown
n./v. 霧氣；以霧籠罩
v. 延期
n. 貪婪
adj./n. 綠色的
n. 航空公司；航線
n. 困境
v. 等候；期待
n./v. 挑戰
adj. 滿的；完全的
n. 緊急
adj. 小心謹慎的
v.(n. ) 指派
n./v. 表演；展現
n. 蝴蝶
Unknown
adj. 自願的
n. 動量；動向
adj. 相當大的
n. 雜物；垃圾
v./n. 集合；開玩笑
adj. 不公平的
Unknown
v. 回復
n./v. 帽子；覆蓋
n. 竹子
Unknown
n. 國家；民族
v. 讀
n. 深思
adj./adv. 生病的
n./v. 躺臥
adv. 在起初
v. 消失
adj. 令人信服的
v. 慫恿
n. 方向
adj. 坐落在…的
adj./adv. 接近的
n. 元首
n. 同伴
n. 敵人
adj. 理性的
Unknown
Unknown
v. 召喚
n. 海岸；沿海地區
n. 油輪
n. 主旨
n. 揚聲器
adj. 勝利的
prep./adv./conj.  在…之前
adj. 不朽的
Unknown
Unknown
adv./pron./conj.  在哪裡；哪裡
v. 減輕(痛苦)
adj. 愚蠢的
v. 藐視
adj. 肺(病)的
n. 一擊
n. 藝術；藝術品
adv. 甚至；連
n./v. 推進；將…提前
n./v. 笑
v. 調查；研究
n/v. 速度；急行
n. 忠誠
n. 羊
n. 情感；激動
n. 身分；同一性
n. 問題
n. 圓盤；唱片
adj. 不禮貌的
n. 文法
n. 權貴
v. 加深
adj./n. 軍事的；軍隊
v. 根除
n. 雨傘
Unknown
v. 綁架
n./v. 打滑
n. 教科書
n. 移民
n. 擴音器
v. 解釋
Unknown
n. 偏愛
v. 削弱
v. 創造
n. 堆
n. 分開
n./v. 等級；並排
adj. 有同情心的
n. 細枝
n. 門
adj. 使震驚
n. 學院院長；主任牧師
adj. 雄偉的；重大的
n. 真空；空虛(感)
adj. 傑出的；卓越的
n./v. 霜；凍
n. 貢獻；捐獻
n. 顧客
adj. 迷信的
n. 攤位
Unknown
n./v. 跳，躍；使反彈
Unknown
v. 說服
n. 垃圾；廢物
Unknown
n. 字母系統
n. 塞子
n. 帶子；樂團
n. 麵粉
n./v. 磨坊；碾碎
n. 海岸巡邏隊
n./v. 跳躍；舞
v. 討論
n./v. 網；用網捕
adj./n. 白色的
Unknown
Unknown
v. 劫取
n./v. 特徵
n. 毛皮
n. 高度
v. 隱約的或威脅性的出現
n. 簽字
n. 目錄
Unknown
prep./adv. 在遠處
n. 商店
n. 慈悲
n. 好奇心
v. 解答
Unknown
n. 檸檬水
adj. 豐富
adj./n. 持異議的(人)
n. 討厭的事物、人、行為等
adj. 受過教育的
n. 圖書館
n./adj. 未來；前途
n./v. 砰然(聲)
v. 防守；防衛
n. 浴缸；盆
n. 心理學
Unknown
n. 鳥
n. 洗衣店
n. 幼稚園
Unknown
n./v. 同意
n. 新來者
adj. 憲法的
n. 電報
n./adj. 平均重量以下；不重要的人
adj. 個人的
adj./adv. 快速的
v./n. 禁止通商
adj./n. 紀念的；紀念品
n. 海灣
Unknown
n. 監護人；警長
Unknown
Unknown
n./v. 點頭；打瞌睡
Unknown
n. 胃口；食慾
n. 清潔工(劑)
adj. 有說服力的
n. 能力
Unknown
adj. 生物(學)的
adj. 無知的
adv. 無論如何
n./v. 衝擊；產生影響
n. 三明治
n. 警察
n. 選民
n. 牡蠣
adv. 酷寒的
adj. 明顯的
n. 棲木；高而安全的地位
n. 估算
pron. 無論是誰
n. 生麵團
adj. 寂靜的
n. 衛生
Unknown
adj. 文雅的
v. 發射
n. 顧問
n. 不幸；悲慘
ad v. 幾乎不
n. 階級組織
n. 英國人
adj. 適當的
n. 協調
n. 財產；好運
n. 狀態
Unknown
n. 指導方針
adv. 笨拙地
n./v. 戰役
n. 組成部分
v. 來到；甦醒
n. 精緻
n. 疾病
Unknown
adv. 無疑地
adj./adv. 每日的
Unknown
adj. 文化的；教養的
Unknown
adj./v. 自己的
v. 趕走
n. 橫切
Unknown
n. 孿生子之一
n./v. 自豪
n. 腫瘤
n./v. 群；團體；聚集
n. 賓果遊戲
n. 突擊
n. 鞋帶；花邊
n. 家庭教師
adj./n. 十九
n. 任務
n. 牙膏
adj. 傑出的
v. 捐贈
adj. 基本的
n. 立場
Unknown
n. 教授
n. 蜻蜓
n. 溫室
adj. 省的
v./n. 落下
prep. 像；和·一樣
adj./n. 空白的
adj./v. 滿足的
v. 想
n. 毫克
n. 現代化
n./v. 預測；預言
n./v. 快照；突然折斷
adj. 優雅的
n./v. 雷聲；打雷
n. 廟宇
n./v. 舞蹈
n. 願望
v. 愛撫
adj. 很可能的
n. 前言
n. 謎題
n. 螞蟻
n. 網
n. 房間；室
n. 大道
Unknown
n. 穩定性
n. 浴缸
adj./n. 邪惡的
n. 海灣；深淵
v. 使能夠
n. 洗禮
n. 領巾
v. 反抗
n. 馬夫
Unknown
v. 期望
adj. 廣大的
n. 設計師
n. 陽光
n./v. 架構
v. 使悲傷
pron. 我們
n. 善良
n. 居民
n. 廢物；垃圾
n. 四分之一
n. [pl. ]腳
adj. 實際的
n. 胡蘿蔔
Unknown
Unknown
adj./n. 八十
n. 馬鞍
n. 附件
Unknown
n./v. 群；聚集
adj. 經濟的；節儉的
Unknown
n. 支撐物
n. 使節團；使命
Unknown
Unknown
n. 狂熱的崇拜
n. 裙子
n. 反抗者
adj. 正常地；普通地
n. 反抗
n./v. 拓荒者；開拓
Unknown
n. 掌聲
v. (使)變寬
n. 總數
n. 輪椅
n. 政策
v. 教導
n. 勇敢；勇氣
adj. 笨拙的
adv. 舒適地
n. 不利之條件
n./v. 旅行
n./v. 恐慌
v. 發癢
adj. 不可預測的
n. 手冊
v. 屈辱
n. 群
Unknown
n. 障礙；缺點
n. 星期三
n./v.  呼叫；打電話
v. 飄浮；搖擺
adj. 有禮貌的
n. 虛偽
n. 鳳梨
v. 表現；行為舉止
n. 教導；指示
n. 公園
n. 廁所
n. 相似
Unknown
v. 變成
Unknown
adj./v./n. 破產的
prep. 每
n./v. 等候
n. 粉末
n. 資料
n. 付費高速公路
n. 悲痛
v./n. 捕獲；獲物
n. 政治集團
adj. 感到羞恥的
n. 口譯者
v. 問候
Unknown
n. 失敗
v. 關聯
n. 部長；外交使節
adj. 空白處的；邊際的
adj./n. 共和國的；共和主義者
v. 使多樣化
adj./v. 完全的；完成
adj. 過時的
Unknown
n. 髮型
adj. 必死的
adj. 看不見的
adj./n. 相反的
n. 少女
n. 缺口；峽谷；隔閡
n. 行人
v. 做；到達；獲得
n./v. 球棒
n. 市長；鎮長
n. 不屈不撓
n. 惡夢
n. 電話
adj./n. 女的；雌性的
Unknown
n. 報告者
n. 義務；稅
adj. 急躁的
n./v. 解放
adj. 好奇的
n. 變換
n. 錯誤；過失
n. 生命
v. 使發出聲
Unknown
adj. 有熱忱的
Unknown
adj. 操作的；管理的
n. 綽號
n. 差別
n. 光線
n. 審判
v./n. 劃分；分歧
n. 單位
n. 偽君子
Unknown
n. 祖國；家夠卜
adv. 其他；另外
adj. 金色的；貴重的
n. 達到
adj. 白尊的
n. 儲存
n. 青少年
n. 公園
n. 記錄
v. 遵守
Unknown
v. 歌唱
n. 假設
adv. 謙虛地
n. 巧合
v.(n. )給～提供娛樂
adj. 天真無邪的
n. 自傳
n. 烤箱
n. 倫理
n./v. 金銀財寶；珍惜
n. 南極或北極
v. 典當
n./v. 停留
v. 攪動
adj. 善於交際地
n./v. 法官
adj. 不重要的
n./v. 偶然；機會
adj. 甜的
n. 姪兒；外甥
v. 畫；拖拉
adj. 看得見的
adj. 對…不感興趣
n. 理髮師
n. 馬鈴薯
n. 代理人；仲介人
n. 大水罐；投手
v. 賺取；贏得
n. 適應；改編(本)
v. 使複雜化
n. 輟學者
n. 樣式
n. 博物館
n. 酒保
v. (n)使驚愕
n. 隆隆聲；景氣
adj. 不須負責任的
v. 忽視
v. 殘留
n./v. 滾動
n. 影像；形像
n./adj. 悲哀；憂鬱
n. 鑽石
v. 詳述；排演
n. 所有權
Unknown
n. 禮貌；習俗
v. 扭傷
Unknown
n. 大學
n. 語言學
n. 民主
n. 季節
Unknown
v. 屈辱
Unknown
n. 發燒；狂熱
n. 演化(論)
n. 環境；圍繞
v. 蹲伏
n. 新聞播報員
prep./adv./conj./n.  曾經；一旦
n. 牛肉
n. 排水系統
v. 去；從事
n./prop. 北京
adj. 明確的；肯定的
n. 習慣
v./n. 結束；完成
n. 稻草；麥桿
adj. 盡責的
Unknown
adj./n.  牙齒的；齒音(的)
n. 類型
adj./advJpron./conj.  任何的；無論什麼
n. 灰泥
n./v. 運動；鍛鍊
n. 蟑蚣卜；油蟲
n. 識別；身分證明
v. 發明；創造
n. 搬行李工人
n./v. 跳躍
n. 一分錢
n. 膝部
n. 小報
adj. 光滑的
Unknown
n. 表面
n. 小巷子；小徑
v. 挑選
n. 快樂
adj./n. 九十
n. 溝通；訊息
n. 障礙
ad v. 痛苦地
adj. 做為…後果的
n./v. 導致；原因
n. 野蠻人
v. 號叫
adj. 令人驚訝的
Unknown
n. 經濟學
adj./n. 棕色的
v. 選舉
n./v. 打盹
adj. 可交換的
adv. 私下地
n. 試金石；(檢驗)標準
n./v. 爭吵
adj. 電的
n. 磁鐵
n. 一長列
Unknown
adj. 心胸寬闊的
n. 蜂蜜
n. 公司；同伴
n./v. 閃爍
Unknown
adj. 政府的
Unknown
adj./pron.  許多
adv. 主要(地)
Unknown
n. 蠟燭
n./v. 罐頭；金屬容器
adv. 有朝一日
adv. 不願意地
n. 國家；鄉下
n. 世代；產生
n. 革新
adj. 聰明的
adj. 擁擠的
n. 假日
v. 拆卸
adj. 戶外的
n./v. 利益；興趣
n. 吸引力
n. 保留
n. 離合器；緊握
adv. 在室內
adv./adj./n. 樓下
n. 探訪者
n. 陽光
adj. 必要的；精華的
adj./n./v. 迷戀
n. 拳擊
v. /n贊助(者)
adv. 相對地
n. 標題
adv. 同樣地
n. 義大利文
v. 不斷地挑剔
adv. 幾乎
n. 工程師
adj. 防護的；保衛的
n. 生物化學
n. 責任
adj. 持久的
v. 描繪
adj. 明顯的
n. 活動；活動力
n. 決心
n. 能力；才能
n. 移民
adj. 每個
adv. 謙虛地
adj. 特大號的
adj. 室內的
n./v. 堆；疊
v. 使再循環；使回收使用
n. 消費
n. 樂觀；樂天
Unknown
n./v. 睡眠
n. 菜單；項目單
n. 競爭者
n. 手臂
adj./adv./n. 足夠的
Unknown
Unknown
Unknown
adj. 切實可行的
n. 鐵路
v./n.  跳水；潛水
n. 對手
n. 回聲；回響
n.  man的複數
n. 雞尾酒
n./v. 土壤；弄髒
Unknown
n. 標籤
v. 與…商量
adj. 能；有能力的
n. 評論家
n. 加侖
adj./adv./pron. 任何的；若干
n./v. 出口
adj. 神的；神聖的
n./v. 主人
v. 自吹自擂；誇耀
n. 瓦解
n. 類型
n. 專欄作家
n. 戲劇
adj. 頭暈目眩的
v./n. 展示
Unknown
n. 比薩
n. 保險
n. 甜點；餐後點心
n. 禮品；天賦
adj. 熱的；熱情的
n. 黏土；泥土
n. 恃強凌弱者
adj. 疼痛的
adj. 磁鐵的；帶磁性的
n./v. 鞦韆；搖擺
v. 發誓；保證
n./v. 榮譽；使…增光
n. 廣告業
adj. 小的
n. 公民；市民
Unknown
n. 通行
v.  分配
v. 縫
v./n. 證明有罪；囚犯
n. 意義；重要性
n. 宴會
n ./v. 交易；處理
v. 循環；流通
n./v. 使混亂；騷亂
n. 暴力
n. 燻豬肉；培根肉
n. 後代；後裔
n./v. 行列；排隊
adj./n. 數字式的
n. 噴射機
n. 抱怨
n. 政府；政體
n. 颶風；暴風雨
n./v. 邊緣；使鋒利
n./v. 口水；吐
v. 證明；展示；遊行
n./v. 姿勢
n. 收到；收據
n. 舌」鬍刀
adj. 違法的
n./v. 草稿
Unknown
n. 侵蝕
Unknown
Unknown
n. 說謊的人
Unknown
n. 百萬
n. 判決
n. 尊嚴
n. 工程
n. 桃子
adv. 幽默地
adj. 成熟的
n. 候選人
n. 出納員
v./n. 猜測
adj./n. 水晶的
n./v. 現金
n. 海軍將官
adj. 英俊的
v. 逃走；逃離
adj./adv. 硬的；艱難的
n. 鯨
n. 相等；平等；均等
n. 廉正
n. 傳統
adj. 寧靜的
